Microsoft.Packaging.Tools/tasks/targets/Microsoft.Packaging.Tools.targets

```diff
@@ -61,6 +61,7 @@
                                 ReferenceCopyLocalPaths="@(ReferenceCopyLocalPaths)"
                                 OtherRuntimeItems="@(_LockFileAssemblies)"
                                 PlatformManifests="@(PackageConflictPlatformManifests)"
+                                ForcedPackages="$(PackageConflictForcedPackages)"
```

Comment:
ItemGroups aren't so easy to control the order: you can't easily prepend, which is why I've kept it a property.